packages/web-react/src/components/UNSTABLE_Toggle/README.md

```diff
@@ -0,0 +1,107 @@
+
+## Basic Usage
+
+The Toggle component implements the HTML [checkbox input][mdn-checkbox] element. It uses
+the native input element and styles it to look like a toggle switch.
+
+```jsx
+import { UNSTABLE_Toggle } from '@lmc-eu/spirit-web-react/components';
+
+<UNSTABLE_Toggle id="toggle-default" label="Toggle Label" />;
+```
+
+## Indicators
+
+If you need to indicate the state of the toggle, you can add the `hasIndicators` prop. This will add a visual indicators to the toggle switch.
+
+```jsx
+<UNSTABLE_Toggle id="toggle-indicators" label="Toggle Label" hasIndicators />
+```
+
+## Required
+
+Add the `isRequired` prop to mark it as required.
+
+```jsx
+<UNSTABLE_Toggle id="toggle-required" label="Toggle Label" isRequired />
+```
+
+## Hidden Label
+
+```jsx
+<UNSTABLE_Toggle id="toggle-hidden-label" label="Toggle Label" isLabelHidden />
+```
+
+## Fluid
+
+```jsx
+<UNSTABLE_Toggle id="toggle-fluid" label="Toggle Label" isFluid />
+```
+
+## Helper Text
+
+```jsx
+<UNSTABLE_Toggle id="toggle-helper-text" label="Toggle Label" helperText="Helper text" />
+```
+
+## Validation States
+
+Validation states can be presented either by prop `validationState`. See Validation state [dictionary][dictionary-validation].
+
+```jsx
+<UNSTABLE_Toggle id="toggle-success" label="Toggle Label" validationState="success" />
+<UNSTABLE_Toggle
+  id="toggle-warning"
+  label="Toggle Label"
+  validationText="Validation text"
+  validationState="warning"
+  isChecked
+/>
+<UNSTABLE_Toggle
+  id="toggle-danger"
+  label="Toggle Label"
+  validationText={['First validation text', 'Second validation text']}
+validationState="danger"
+/>
+```
+
+## Disabled State
+
+On top of adding the `isDisabled` props to disable Toggle.
+
+```jsx
+<UNSTABLE_Toggle id="toggle-disabled" label="Toggle Label" isDisabled />
+```
+
+## API
+
+| Name              | Type                                           | Default | Required | Description                    |
+| ----------------- | ---------------------------------------------- | ------- | -------- | ------------------------------ |
+| `id`              | string                                         | -       | ✓        | Input and label identification |
+| `isDisabled`      | boolean                                        | `false` | ✕        | Whether is toggle disabled     |
+| `isChecked`       | boolean                                        | `false` | ✕        | Whether is toggle checked      |
+| `isLabelHidden`   | boolean                                        | `false` | ✕        | Whether is label hidden        |
+| `isFluid`         | boolean                                        | `false` | ✕        | Whether is toggle fluid        |
+| `hasIndicators`   | boolean                                        | `false` | ✕        | Whether has visual indicators  |
+| `label`           | string                                         | -       | ✕        | Label text                     |
```

Comment:
Actually it is. I only forgot to mark it in README. Thanks

Reply:
Then it should be required in the other components as well… 👀 

→ https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/src/components/Checkbox/README.md?plain=1#L40
→ https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/src/components/TextArea/README.md?plain=1#L49
→ https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/src/components/Select/README.md?plain=1#L46
→ https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/src/components/Radio/README.md?plain=1#L38

Reply:
BC?

Reply:
In the `TextArea` and `Select` it would be BC.

In the case of `Checkbox` and `Radio` it looks like there might be inconsistence between code and documentation, so probably in the code is `label` required, but it is not mentioned in the README. Create new issue?

Reply:
Maybe let's discuss it 🤔 @crishpeen @literat @pavelklibani ?

Reply:
I agree with solving this in a separate issue.

Reply:
https://jira.almacareer.tech/browse/DS-1396 created